components/constellation/constellation.rs

```diff
@@ -2854,7 +2852,7 @@ where
         thread_name: Option<String>,
         entry: LogEntry,
     ) {
-        debug!("Received log entry {:?}.", entry);
+        // trace!("Received log entry {:?}.", entry);
```

Comment:
Same comment here about commented out code.

Reply:
done in 9772e8a36bd55a3619c20199e5081cfee8491cab